spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+RSpec.describe do
+  before(:each) do
+    @facility_factory = FacilityFactory.new
+    @facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014'})
```

Comment:
You don't need this. The only thing that should be in your setup is
```ruby
@facility_factory = FacilityFactory.new
@or_dmv_office_locations = DmvDataService.new.or_dmv_office_locations
```
If you want to use the `before(:each)` then you will want to put the other states in there as well
```ruby
@new_york_facilities = DmvDataService.new.ny_dmv_office_locations
@missouri_facilities = DmvDataService.new.mo_dmv_office_locations
```
Then you would just write separate tests for each state. 
We don't want to pass `@facility_1` info into line 7. 
So our expect statements should be
```ruby
expect(@facility_factory.create_facilities(@or_dmv_office_locations)).to be_an(Array)
```
Note the expect statement would change based on the state data you are passing to `@facility_factory.create_facilities`.